### ISSUE TITLE
Can we have something like Detox extension?

### ISSUE THREAD

**Issue by ikhthiandor** [The-Compiler/qutebrowser#1296]
Website: http://usedetox.com/

It would be very nice to have. This is the only thing I'm missing constantly from Firefox right now. If it's not possible right now how long before we can have extensions like this in Qutebrowser? Any relevant issues or pr I can look at?

**Comment by lahwaacz** [comment]
If it only modifies the web page and not the browser's interface, it should be possible to implement it as a Greasemonkey userscript. See #664

**Comment by The-Compiler** [comment]
What @lahwaacz said - unfortunately that PR isn't fully finished and the author doesn't answer anymore. From what its looking like, however, with the future backend (QtWebEngine, see #666) this should be easier to implement.

Also see #30 for general ideas about plugin support - however it'll probably take some months until I get the time to work on that. I added a comment there, so I'm closing this.
